apps/application-system/api/src/app/modules/application/tools/applicationAdmin.serializer.ts

```diff
@@ -260,6 +260,15 @@ export class ApplicationAdminStatisticsSerializer
   }
 
   async serializeArray(applications: ApplicationsStatistics[], locale: Locale) {
-    return Promise.all(applications.map((item) => this.serialize(item, locale)))
+    return (
+      await Promise.allSettled(
+        applications.map((item) => this.serialize(item, locale)),
+      )
+    )
+      .filter(
+        (item): item is PromiseFulfilledResult<Record<string, any>> =>
+          item.status === 'fulfilled',
+      )
+      .map((item) => item.value)
```

Comment:
Getur this.serialize return-að item.value svo það þurfi ekki að map-a -> filter-a -> map-a aftur?

Reply:
`item.value` er skilagildið úr [Promise.allSettled()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) og týpan er því `PromiseFulfilledResult`